lumicks/pylake/fdcurve.py

```diff
@@ -26,6 +31,21 @@ class FdCurve(DownsampledFD):
         The time range (ns) of this FD curve within the file.
     name : str
         The name of this FD curve as it appeared in the timeline.
+
+    Examples
+    --------
+    ::
+
+        import lumicks.pylake as lk
+        f = lk.File("fdcurves.h5")
+
+        # Show FdCurve "1"
+        fd_curve = f.fdcurves["1"]
+        fd_curve.plot()
+
+        # Make a new F,d curve based on the `start` and `stop` time of FdCurve "1", but only include
+        # the x-component of the force on trap 2
+        fd_curve = fd_curve.with_channels("2x", distance="1")
```

Comment:
I wonder if it's better to move the example section before attributes. Only because, in the actual rendering at first glance it's difficult to tell that the example is part of the class docstring
![image](https://user-images.githubusercontent.com/61475504/231159730-836b235b-bd08-46bf-8ca0-1a975dfb0288.png)

Reply:
Yeah, that's a fair point. I swapped it around.